styles/style.css

```diff
@@ -0,0 +1,170 @@
+/* Global styles
+--------------------------------*/
+
+body {
+    color: #343434;
+    margin: 0;
+    padding: 0;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 15px;
+    line-height: 1.5;
+}
+
+img {
+    width: 300px;
+}
+
+a {
+    color:rgba(41,86,78,1);
+}
+
+a:hover {
+    text-decoration: none;
+    color: magenta;
+}
+
+h1, h2 {
+    font-family: 'Caveat', cursive;
+    font-size: 45px;
+    margin: 0;
+}
+
+.content-wrap {
+    max-width: 800px;
+    width: 85%;
+    margin: 0 auto;
+    padding: 10px 0;
+    display:grid;   
+}
+
+h3 {
+    margin-bottom: 0;
+    color : teal
+}
+
+.item-details h3 + p {
+    font-style: italic;
+}
+
+.item-details h3 ~ p {
+    margin: 0;
+}
+
+.divider > section {
+    border-bottom: 1px dashed #343434;
+    padding: 25px 0;
+}
+
+.divider > section:last-of-type {
+    border-bottom: none;
+
+}
+
+.picture {
+    margin-top: 15px;
+    margin-bottom: 15px;
+}
+
+
+/* profile
+--------------------------------*/
+header {
+    background:rgba(107,223,204,0.8);
+    color:white;
+}
+
+/* projects
+--------------------------------*/
+.projects {
+    background: #F7FFF7;
+}
+
+.projects .btn {
+    color: #F7FFF7;
+    background: rgba(4,174,145,1);
+    text-decoration: none;
+    padding: 8px;
+    border-radius: 4px;
+    display: inline-block;
+}
+.projects .btn:hover {
+    background: rgb(47, 48, 97, 80%);
+}
+
+.project-item {
+    overflow: hidden;   
+}
+
+.project-item h3 {
+    margin-top: 0;
+}
+/* work experience
+--------------------------------*/
+
+.work-experience {
+    background:rgba(188,245,235,0.8);
+}
+
+/* Education
+--------------------------------*/
+.education {
+    padding-bottom: 100px;
+}
+
+/* contact Info
+--------------------------------*/
+footer {
+    background: rgba(107,223,204,0.8) ;
+    color: #F7FFF7  ;
+}
+
+.contact-list ,
+.nav-bar {
+    list-style-type: none;
+}
+
+.contact-list a ,
+.nav-bar a {
+    padding: 15px;
+    display: inline-block; 
+}
+
+
+
+
+/* Reponsive
+--------------------------------*/
+@media screen and (min-width:750px){
+    header, footer {
+        text-align: center;
+    }
+    .project-item img {
+        float:left;
+        margin-right: 20px;
+    }
+    .job-item {
+        display: grid;
+        grid-template-columns: 1fr 2fr;
+        column-gap: 150px;
+    }
+    .contact-list {
+    display: flex;
+    justify-content: center;
+    }
+    .nav-bar {
+        display: flex;
+        justify-content: right;
+        }
```

Comment:
Try to be consistent in your CSS code style. Some elements have different levels of indentation/spacing which can impact readability.